ui/app/models/pki/issuer.js

```diff
@@ -69,14 +88,32 @@ export default class PkiIssuerModel extends PkiCertificateBaseModel {
   usage;
 
   @attr('string', {
-    label: 'Manual chain',
     subText:
       "An advanced field useful when automatic chain building isn't desired. The first element must be the present issuer's reference.",
   })
   manualChain;
 
+  @attr({
+    subText:
+      'The signature algorithm to use when building CRLs. The default value (empty string) is for Go to select the signature algorithm automatically, which may not always work.',
+    noDefault: true,
+    possibleValues: [
+      'sha256withrsa',
```

Comment:
Do these need to be capitalized like the [golang values](https://pkg.go.dev/crypto/x509#SignatureAlgorithm)?

Reply:
I had that initially but then when testing got an API error and these were the accepted values 🤷‍♀️